capstone/src/main/webapp/youtubeScript.js

```diff
@@ -162,15 +202,83 @@ function showAvailableAttributes() {
     label.appendChild(document.createTextNode(attribute));
     avaiableAttributesElement.appendChild(checkbox);
     avaiableAttributesElement.appendChild(label);
-    avaiableAttributesElement.appendChild (document.createTextNode (" "));
+    avaiableAttributesElement.appendChild(document.createTextNode (" "));
   });
 }
 
 /** Checks if a character is a letter */
 function isLetter(character) {
-  if ((character.charCodeAt() >= 65 && character.charCodeAt() <= 90) || (character.charCodeAt() >= 97 && character.charCodeAt() <= 122)) {
-    return true;
-  } else {
-    return false;
+  return (character.charCodeAt() >= 65 && character.charCodeAt() <= 90) || (character.charCodeAt() >= 97 && character.charCodeAt() <= 122); 
+}
+
+async function getTrending(categoryId) {
+  const trendingResponse = await fetch('/trending_servlet?videoCategoryId=' + categoryId);
+  const trendingResponseJson = await trendingResponse.json();
+  const trendingVideoIds = [];
+  for (const item in trendingResponseJson.items) {
+    const videoId = trendingResponseJson.items[item].id;
+    trendingVideoIds.push(videoId);
+  }
+  const commentsList = []
+  for (const id in trendingVideoIds) {
+    const videoCommentList = await fetch('/youtube_servlet?videoId=' + trendingVideoIds[id]);
+    const videoCommentListJson = await videoCommentList.json();
+    commentsList.push(videoCommentListJson);
+  }
+  inputCommentsToPerspective(commentsList);
+}
+
+function enableTextInput(button) {
+  if (button.checked) { 
+    document.getElementById('channelIdForAnalysis').value = button.id;
+    document.getElementById('channelIdForAnalysis').disabled = true;   
+  }
+}
+
+function disableTextInput(button) {
+  if (button.checked) { 
+    document.getElementById('channelIdForAnalysis').value = "";
+    document.getElementById('channelIdForAnalysis').disabled = false;
+  }
+}
```

Comment:
nit: These two methods could be condensed into one and you could pass in a `value` and `isDisabled` parameter to avoid code repetition

Reply:
addressed in "consolidate text input toggle function"